client/src/index.js

```diff
@@ -1,39 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
 import Home from "./components/Home";
 import "semantic-ui-css/semantic.min.css";
 import About from "./components/About";
 import Status from "./components/Status";
-import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import { NavLink, BrowserRouter as Router, Route } from "react-router-dom";
+import { Menu, Container, Image, Input } from "semantic-ui-react";
 
 const Routes = () => {
   return (
     <Router>
-      <div>
-        <ul>
-          <li>
-            <Link className="nav-link" to="/">
-              Home
-            </Link>
-          </li>
-          <li>
-            <Link className="nav-link" to="/about">
-              About
-            </Link>
-          </li>
-          <li>
-            <Link className="nav-link" to="/status">
-              Status
-            </Link>
-          </li>
-        </ul>
+      <Menu inverted>
+        <Container>
+          <Menu.Item header>
+            <Image
+              size="small"
+              src="http://www.maryhillintegration.org.uk/wp-content/uploads/2016/10/banner.jpg"
+            />
+          </Menu.Item>
+
+          <Menu.Item header as={NavLink} to="/" name="home" />
+          <Menu.Item header as={NavLink} to="/about" name="about" />
+          <Menu.Item header as={NavLink} to="/status" name="status" />
```

Comment:
```suggestion
          <Menu.Item as={NavLink} to="/" name="home" />
          <Menu.Item as={NavLink} to="/about" name="about" />
          <Menu.Item as={NavLink} to="/status" name="status" />
```
It looks like `header` isn't bolding the text because we are rendering a `NavLink` instead, so we can remove the `header` attribute from these three lines since it's not doing anything

Reply:
Ok i am gonna remove them. Thank you